docs/pages/versions/unversioned/sdk/filesystem.mdx

```diff
@@ -11,14 +11,32 @@ isNew: true
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 
-`expo-file-system` provides access to the file system stored locally on the device. It can also download files from the network.
+`expo-file-system` provides access to files and directories stored on a device or bundled as assets into the native project. It also allows downloading files from the network.
 
 ## Installation
 
 <APIInstallSection />
 
 ## Usage
 
+```js
+import { File, Directory, Paths } from 'expo-file-system';
+```
+
+The `File` and `Directory` instances hold a reference to a file, content, or asset URI.
+
+The file or directory does not need to exist &mdash; an error will be thrown from the constructor only if the wrong class is used to represent an existing path (so if you try to create a `File` instance passing a path to an already existing directory).
+
+## Features
+
+- Both synchronous and asynchronous, read and write access to file contents
+- Creation, modification and deletion
+- Available properties, such as `type`, `size`, `creationDate`, and more
+- Ability to read and write files as streams or using the `FileHandle` class
+- Easy file download/upload using `downloadFileAsync` or `expo/fetch`
+
+## Examples
+
 ### Writing and reading text files
```

Comment:
it'd be nice to put these examples under Collapsibles so that it's easier to scroll/scan the docs